android/src/main/java/com/applovin/reactnative/AppLovinMAXModule.java

```diff
@@ -954,9 +954,16 @@ public void setBannerExtraParameter(final String adUnitId, final String key, fin
     }
 
     @ReactMethod
-    public void setBannerLocalExtraParameter(final String adUnitId, final String key, final String value)
+    public void setBannerLocalExtraParameter(final String adUnitId, final ReadableMap object)
```

Comment:
Let's name the parameter accordingly, it's a map containing the object we care about so I would just call it `parameterMap`.

Reply:
Renamed parameterMap in Android and parameterDict in iOS.